AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Models/AWSAuthCognitoSession.swift

```diff
@@ -77,6 +78,24 @@ public struct AWSAuthCognitoSession: AuthSession,
 
 }
 
+/// Internal Helpers for managing session tokens
+internal extension AWSAuthCognitoSession {
+    func areTokensExpiring(in seconds: TimeInterval? = nil) -> Bool {
+        
+        guard let tokens = try? cognitoTokensResult.get(),
```

Comment:
Can we add these under "cognitoUserPoolTokens" struct? So that we do not need to do the condition like below

Reply:
we can do it, in that case the method would need to be public, because the tokens model exists in the `AmplifyPluginsCore` .. I wasn't sure if we really needed that logic to be public.. Thats the reason I had it here..

Reply:
we could add an internal extension for the struct in the plugin right?